packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/DartMessenger.java

```diff
@@ -77,13 +44,17 @@ enum CameraEventType {
    */
   public void sendDeviceOrientationChangeEvent(
       @NonNull PlatformChannel.DeviceOrientation orientation) {
-    this.send(
-        DeviceEventType.ORIENTATION_CHANGED,
-        new HashMap<String, Object>() {
-          {
-            put("orientation", CameraUtils.serializeDeviceOrientation(orientation));
-          }
-        });
+    handler.post(
+        () ->
+            globalEventApi.deviceOrientationChanged(
+                CameraUtils.orientationToPigeon(orientation),
+                new Messages.VoidResult() {
```

Comment:
Let's use [this pattern](https://github.com/flutter/packages/blob/main/packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/NoOpVoidResult.java) of consolidating the no-op handlers into one place with an explanation. You can just directly copy that file into this package.